# substitute for gringo.Fun
string robot
string action
